scripts/check_compliance.py

```diff
@@ -737,17 +739,18 @@ def run(self):
             .splitlines()
 
         # Filter out everything but Python files
-        py_files = filter_py(files)
+        py_files = filter_py(GIT_TOP, files)
         if not py_files:
             return
 
+        pylintcmd = ["pylint", "--rcfile=" + pylintrc] + py_files
+        logger.info(" ".join([shlex.quote(word) for word in pylintcmd]))
         try:
             # Run pylint on added/modified Python files
-            process = subprocess.Popen(
-                ["pylint", "--rcfile=" + pylintrc] + py_files,
+            process = subprocess.Popen(pylintcmd,
```

Comment:
`pylintcmd` should go on a separate line, so that the second argument isn't less indented than the first argument.